starknet-rpc-test/src/lib.rs

```diff
@@ -140,29 +140,46 @@ fn get_free_port() -> Result<u16, TestError> {
     Err(TestError::NoFreePorts)
 }
 
+fn get_free_p2p_port() -> Result<u16, TestError> {
```

Comment:
why do we need to set the p2p port? it's setted as a random one if the default is already occupied and we are not querying it